test/GPv2Settlement/ComputeTradeExecutions.ExecutedAmounts.t.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8.26;
+
+import {BaseComputeTradeExecutions, GPv2Order, SettlementEncoder} from "./ComputeTradeExecutions.t.sol";
+
+contract OrderExecutedAmounts is BaseComputeTradeExecutions {
+    using GPv2Order for GPv2Order.Data;
+    using SettlementEncoder for SettlementEncoder.State;
+
+    function test_should_compute_amounts_for_fill_or_kill_sell_orders() public {
+        GPv2Order.Data memory order = defaultOrder();
+        order.kind = GPv2Order.KIND_SELL;
```

Comment:
The test explicitly says "fill or kill", I don't think the reader should be required to check that the "default order" is fill or kill. This also applies to all subsequent tests.

```suggestion
        order.kind = GPv2Order.KIND_SELL;
        order.partiallyFillable = false;
```